### PR TITLE
[luci-interpreter] Enable Square

### PR DESCRIPTION
**Pull request by seanshpark**
This will enable Square operator.

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
For #7067 , from draft #7071

File: compiler/luci-interpreter/src/kernels/Square.cpp [hunk 1]
```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 3021 Samsung Electronics Co., Ltd. All Rights Reserved
```
Comment (jinevening) [inline compiler/luci-interpreter/src/kernels/Square.cpp:2]:
```suggestion
 * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
```

File: compiler/luci-interpreter/src/kernels/CMakeLists.txt [hunk 2]
```diff
@@ -115,6 +115,8 @@ set(SOURCES
     StridedSlice.cpp
     Sqrt.h
     Sqrt.cpp
+    Square.h
+    Square.cpp
```
Comment (jinevening) [inline compiler/luci-interpreter/src/kernels/CMakeLists.txt:119]: Please add `Sqaure.test.cpp` in the same file.

Comment (seanshpark) [reply compiler/luci-interpreter/src/kernels/CMakeLists.txt:119]: Thanks! forgot to copy-paste -_-;

**Review by jinevening — APPROVED** [review]
LGTM

**Pull request merged**